src/Normalizer/Normalizer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Http\Client\Plugin\Normalizer;
+
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Normalize a request or a response into a string or an array
```

Comment:
oh, i see. hm. it does make sense to inject the string converter. but it will be a BC break when we start a separate tool and switch to that. could we for now simply instantiate Normalizer in the plugin and not have an argument for Normalizer? then we can add an optional argument once we have built the separate package to convert requests to strings, without creating a BC break for users.